### PR TITLE
PIL-593 Accessibility

### PR DESCRIPTION
**Pull request by OrangeBrid**

### CODE REVIEW

File: app/views/TaskListView.scala.html [hunk 1]
```diff
@@ -75,25 +75,25 @@ <h2 class="app-task-list__section">
...
-                        <span class="hmrc-status-tag"  id="submit-pay-status-co">
+                        <span class="hmrc-status-tag"  id="submit-pay-status-co nfm-status">
                             @messages("task.status.notStarted")</span>
                         }
                     </li>
                     } else{
                     <li class="app-task-list__item">
-                        <span class="app-task-list__task-name">
+                        <span class="app-task-list__task-name" id="nfm-status">
                             @messages("taskList.task.business.filingMember")
```
Comment (jassalrichy) [inline app/views/TaskListView.scala.html:89]: duplicate id line 88 and (92/94/96)

File: app/views/TaskListView.scala.html [hunk 2]
```diff
@@ -102,25 +102,25 @@ <h2 class="app-task-list__section">
...
+                                <a href="/report-pillar2-top-up-taxes/further-details/group-status" aria-describedby="group-details-status">
                                  @messages("taskList.task.business.sub.add")
                                 </a>
                             } else {
                              @messages("taskList.task.business.more")
                             }
                         </span>
                         @if(filingMemberStatus == "Completed" && groupDetailStatus=="Completed" ){
-                        <span class="hmrc-status-tag" > @messages("task.status.completed") </span>
+                        <span class="hmrc-status-tag" id="group-details-status" > @messages("task.status.completed") </span>
```
Comment (jassalrichy) [inline app/views/TaskListView.scala.html:117]: duplicate id: group-details-status line 105 & 117 ?

File: app/views/TaskListView.scala.html [hunk 3]
```diff
@@ -75,25 +75,25 @@ <h2 class="app-task-list__section">
...
                         @if(filingMemberStatus == "Completed"){
-                        <span class="hmrc-status-tag" > @messages("task.status.completed") </span>
+                        <span class="hmrc-status-tag" id="nfm-status" > @messages("task.status.completed") </span>
                         } else if(filingMemberStatus == "InProgress") {
-                        <span class="hmrc-status-tag"> @messages("task.status.inProgress") </span>
+                        <span class="hmrc-status-tag" id="nfm-status"> @messages("task.status.inProgress") </span>
                         } else  {
-                        <span class="hmrc-status-tag"  id="submit-pay-status-co">
+                        <span class="hmrc-status-tag"  id="submit-pay-status-co nfm-status">
                             @messages("task.status.notStarted")</span>
```
[the remark this answers is not in the document]
Comment (OrangeBrid) [reply app/views/TaskListView.scala.html:83]: Those are two different id's though? Won't that affect it's function? (Same with other/similar instances)
They ref the aria part
 <... aria-describedby="nfm-status">

[the remark this answers is not in the document]
Comment (jassalrichy) [reply app/views/TaskListView.scala.html:83]: The id name must not contain whitespaces (spaces, tabs, etc.).

[the remark this answers is not in the document]
Comment (jassalrichy) [reply app/views/TaskListView.scala.html:83]: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

[the remark this answers is not in the document]
Comment (OrangeBrid) [reply app/views/TaskListView.scala.html:83]: Ah okay cool. I'll update the tags

File: app/views/TaskListView.scala.html [hunk 4]
```diff
@@ -34,140 +34,57 @@ <h1 class="govuk-heading-l">@messages("task-list.heading")</h1>
...
+                @messages("taskList.task.business.heading")
+            </h2>
+            <ul class="app-task-list__items">
+                    <!-- Ultimate Parent Task -->
+                <li class="app-task-list__item">
+                    <span class="app-task-list__task-name">
+                    @ultimateParentInfo.link.map { link =>
+                    <a href="@link" aria-describedby="eligibility-status">
+                    @messages(s"taskList.task.business.ultimate.${if(ultimateParentInfo.status == "Completed") "edit" else "add"}")
+                    </a>
```
Comment (jassalrichy) [inline app/views/TaskListView.scala.html]: Could create enums for the task status (Completed, CannotStartYet, etc)

**Review by jassalrichy — APPROVED** [review]

**Review by alikouhestani — APPROVED** [review]

**Pull request merged**